package/Codegen/TSEnums.cs

```diff
@@ -18,8 +18,8 @@ namespace Codegen
     /// Class to produce the template output
     /// </summary>
     
-    #line 1 "F:\react-native-xaml\package\Codegen\TSEnums.tt"
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
+    #line 1 "C:\Users\asklar\source\repos\react-native-xaml\package\Codegen\TSEnums.tt"

```

Comment:
Is this the right path? Looks like it's local to your computer.

Reply:
yes, these files are autogenerated, see https://github.com/microsoft/react-native-xaml/pull/171#discussion_r789818292